src/pages/SatelliteManifestPage/__tests__/SatelliteManifestPage.test.tsx

```diff
@@ -39,22 +52,113 @@ describe('Satellite Manifests Page', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('renders correctly with no results', () => {
+  it('renders loading when it has not received a response back', async () => {
     window.insights = {};
+
+    const { authenticateUser } = PlatformServices;
+
+    (authenticateUser as jest.Mock).mockReturnValue(
+      Promise.resolve({
+        identity: {
+          user: {
+            is_org_admin: true
+          }
+        }
+      })
+    );
```

Comment:
Since you are using this same mock repeatedly, you should pull it out into a before block for the whole outer describe block and assign an `isOrgAdmin` variable to `is_org_admin`.  Then add inner describe blocks for each of the states of `isOrgAdmin` in which you assign the applicable value to it.

Reply:
OK, I pushed a tweak to this; I didn't do it exactly as you described -- there is a scoping issue using beforeEach with mocking the authenticateUser function, since we need to check that authenticateUser has been called for these tests.  I created a helper function, mockAuthenticateUser, that returns the authenticateUser mock; this achieves the same thing of cleaning up code repetition and is readable.  LMK thoughts.

Reply:
That's good for now.